dolphin/processor/config.py

```diff
@@ -245,9 +245,18 @@ def get_kwargs_likelihood(self):
             'check_positive_flux': True,
             'check_bounds': True,
             'bands_compute': [True] * self.band_number,
-            'image_likelihood_mask_list': self.get_masks()
+            'image_likelihood_mask_list': self.get_masks(),
+            'prior_lens': []
```

Comment:
I first only added prior_lens, so my code can be check before adding the other paramters. Just to double confirm, I need : 
- 'prior_lens'
- 'prior_lens_kde' 
- 'prior_lens_lognormal',
- 'prior_lens_light'
- 'prior_lens_light_kde',
- 'prior_lens_light_lognormal'

Reply:
Just add `prior_lens`, `prior_lens_light`, `prior_ps`, and `prior_source` for now.